### PR TITLE
IOD-5 view license

### PR DESCRIPTION
**Pull request by aideninbcgov**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/pages/licensees/index.jsx [hunk 1]
```diff
@@ -11,23 +11,23 @@ export default function Licensees() {
...
-			return (
-				<IconButton
-					color='primary'
-					aria-label='go to details'
-					onClick={() => router.push(`/licensees/${params.row.id}`)}>
-					<MoreHorizIcon color='action' />
-				</IconButton>
-			);
-		},
+		// field: 'actions',
```
Comment (akroon3r) [inline src/pages/licensees/index.jsx:14]: do we need to keep this block?

File: src/styles/scss/components/_forms.scss [hunk 2]
```diff
@@ -183,3 +183,85 @@
...
+
+.css-g1d714-ValueContainer {
+  padding: 0px 3px;
+}
+
+.css-b62m3t-container> * {
+  line-height: 1.5 !important;
+  font-size: 0.85rem !important;
+  font-family: var(--body-font-family) !important;
+}
```
Comment (akroon3r) [inline src/styles/scss/components/_forms.scss:267]: add a new line to end of line 267 here please :)

Comment (akroon3r) [reply src/styles/scss/components/_forms.scss:267]:
```suggestion
}

```

Comment (akroon3r) [reply src/styles/scss/components/_forms.scss:267]: ^^ you can commit the suggestion here

File: src/pages/licensees/index.jsx (hunk 1 above)
Comment (aideninbcgov) [reply src/pages/licensees/index.jsx:14]: for now yes. I'd like to discuss with Scott before removing it completely.

Comment (akroon3r) [inline src/pages/licensees/index.jsx:14]:
```suggestion
                // TODO: reminder to talk to Scott about line 14-26
		// field: 'actions',
```

**Review by akroon3r — APPROVED** [review]
sorry for the delay Aiden.

**Pull request merged**